specification/devops/resource-manager/Microsoft.DevOps/preview/2020-07-13-preview/devops.json

```diff
@@ -677,9 +727,9 @@
       "type": "object",
       "description": "Configuration used to bootstrap a Pipeline.",
       "properties": {
-        "repository": {
+        "sourceRepository": {
```

Comment:
As @akning-ms mentioned, this change constitute as "remove" a property from last api-version. ("remove" then "add" a new one).

Though I am not sure how we treat this. The whole RP is in preview version. And this is cross api-version.

Reply:
@weidongxu-microsoft is there an action item on me for this?

Reply:
@weidongxu-microsoft @PhoenixHe-msft What is the action item on me for this? or do i need to wait for a set of reviewers to do the re-review?

Reply:
Also it's a removal and addition of property in a new API version which is still in preview. We are still supporting that property in the old API version, I don't think it should be considered a breaking change due to this

Reply:
+ @JeffreyRichter for guidance on change of property name cross api-version on preview.

Reply:
It is OK to add a property in a preview and then rename it during the preview. Our recommendation is that you version your previews. So, in api-version=2020-12-23-preview.1 you add a property. Then in api-version=2020-12-23-preview.2 you rename the property. If you follow this versioning scheme, then our tooling will not flag these changes as breaking. If you make these changes in the same api-version, then our tooling WILL flag these as breaking.

Also, when you release a new version, you can stop supporting the old preview version in 90 days. This gives your preview customers 90 days to move to the new preview without breaking them all of a sudden which can be frustrating to the customers that are trying to help you with giving you feedback on the preview(s).

Reply:
Since you're in preview, I'll approve this PR but you have to notify your preview customers of the change, how it will impact them, and what timeframe they have to update their code by.